src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,97 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import duke.util.ParsedData;
+
+/**
+ * Parent task class for all other Tasks.
+ */
+public abstract class Task implements Comparable<Task> {
+    protected final String description;
+    protected boolean completed;
```

Comment:
Boolean variables/methods should be named to sound like booleans. Use `hasComplete` or `isComplete` instead.

Reply:
updated to fit coding standard.